packages/amplify-cli-core/src/jsonUtilities.ts

```diff
@@ -39,12 +42,12 @@ export class JSONUtilities {
 
   public static writeJson = (
     fileName: string,
-    data: any,
+    data: unknown,
     options?: {
-      keepComments?: boolean;
       mode?: number;
       minify?: boolean;
       secureFile?: boolean;
+      stable?: boolean; // if true, will print object keys in alphabetical order
```

Comment:
We don't have code that depends on the order per-se but switching the order creates unnecessary git diffs in the customer's repo (and now we have some e2e tests that check for expected diffs and this was failing because the save was reordering stuff). I would argue we should go the other way and start serializing all of our json files in a stable way.

Reply:
let's be brave then. we have to do it eventually, right ?

Reply:
I don't want to change all our file serialization in this PR. I can make a follow up task for it though